<launch>
  <!-- The package has 2 examples of converting joystick commands to servo commands.
       This is the higher-latency, Python-based teleop_tools example.
       You can modify it in your own package for your own type of gamepad or joystick.
       We do plan to accept pull requests of config files for other controller types using this method.
  <remap from="/joint_states" to="/j2n6s300/joint_states"/>
  
  -->

  <!-- This node does the servoing calculations -->
  <node name="servo_server" pkg="moveit_servo" type="servo_server" output="screen" >
    <rosparam command="load" file="$(find joining_experiment)/config/servo_config.yaml" />
    <remap from="/joint_states" to="/j2n6s300/joint_states"/>
  </node>

  <!--<node name="track_target" pkg="joining_experiment" type="get_target.py" output="screen" ></node>-->
  <node name="track_finger" pkg="joining_experiment" type="finger_pos.py" output="screen" ></node>

<!--
  <node name="pose_tracking_example" pkg="moveit_servo" type="pose_tracking_example" output="screen" >
    <rosparam command="load" file="$(find joining_experiment)/config/servo_config.yaml" />
    <rosparam command="load" file="$(find moveit_servo)/config/pose_tracking_settings.yaml" />
  </node>


  <node name="joint_angles" pkg="joining_experiment" type="joint_publisher.py" output="screen" >  </node>


  <node name="joining_pose_tracking" pkg="joining_experiment" type="joining_pose_tracking" output="screen" >
    <rosparam command="load" file="$(find joining_experiment)/config/servo_config.yaml" />
    <rosparam command="load" file="$(find moveit_servo)/config/pose_tracking_settings.yaml" />
    <remap from="/joining_pose_tracking/delta_twist_cmds" to="/servo_server/delta_twist_cmds"/>

  </node>

-->



</launch>
